pkg/agent/billing/billing.go

```diff
@@ -57,6 +58,10 @@ func (m *metricsTimeSlice) Duration() time.Duration { return m.endTime.Sub(m.sta
 type vmMetricsInstant struct {
 	// cpu stores the cpu allocation at a particular instant.
 	cpu vmapi.MilliCPU
+	// number of bytes received by the VM from the open internet since the last time slice
+	ingressBytes vmapi.NetworkBytes
+	// number of bytes sent by the VM to the open internet since the last time slice
+	egressBytes vmapi.NetworkBytes
 }
```

Comment:
FYI: adding `ingressBytes` and `egressBytes` here doesn't do anything unless they're used in `drainEnqueue` to add `billing.IncrementalEvent`s to the queue for processing.

Reply:
Good point, added it there